File: Modul 5/soal 4/PRAK404-2210817220020-Helena Rosalina.c
#include <stdio.h>
int main(){
int p;
double a, b;
char pilihan;
do{
 printf("Pilih program\n");
 printf("1. Penjumlahan\n");
 printf("2. Pengurangan\n");
 printf("3. Perkalian\n");
 printf("4. Pembagian\n");
 printf("5. Exit\n");

printf("Masukkan Pilihan : ");
        scanf("%d", &p);
 if (p > 5 ) {

 } else if (p == 5){
 printf("Terimakasih, telah menggunakan kalkulator HELENA ROSALINA");
 break;
 } else {
 printf("Masukkan Nilai Pertama : ");
 scanf("%lf", &a);
 printf("Masukkan Nilai Kedua : ");
 scanf("%lf", &b);
 }

switch (p)
{
case 1:
printf("Hasil Penjumlahan antara %.2lf dengan %.2lf adalah %.2lf",a,b,a+b);
break;
case 2:
printf("Hasil Pengurangan antara %.2lf dengan %.2lf adalah %.2lf",a,b,a-b);
break;
case 3:
printf("Hasil Perkalian antara %.2lf dengan %.2lf adalah %.2lf",a,b,a*b);
break;
case 4:
if (b == 0)
printf("\n nilai kedua tidak boleh 0");
else
printf("Hasil pembagian antara %.2lf dengan %.2lf adalah %.2lf",a,b,a/b);
break;
default:
printf("Input anda salah, silahkan coba lagi");
 break;
}
printf("\n continue? (y/n)");
scanf(" %c", &pilihan);
} while (pilihan == 'y');
return 0;
}

File: Modul 5/soal 4/PRAK404-2210817220020-Helena Rosalina.py
while True:

    choice = input("Pilih Program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\nMasukkan Pilihan :")
 
    if choice == '1':
        i1 = float(input("Masukkan nilai pertama : "))
        i2 = float(input("Masukkan nilai kedua : "))
        sum = i1 + i2
        print("Hasil penjumlahan adalah ",sum)
    elif choice == '2':
        i1 = float(input("Masukkan nilai pertama : "))
        i2 = float(input("Masukkan nilai kedua : "))
        diff = i1 - i2
        print("Hasil pengurangan adalah ",diff)
 
    elif choice == '3':
        i1 = float(input("Masukkan nilai pertama : "))
        i2 = float(input("Masukkan nilai kedua : "))
        prod = i1 * i2
        print("Hasil perkalian adalah ",prod)
    elif choice == '4':
        i1 = float(input("Masukkan nilai pertama : "))
        i2 = float(input("Masukkan nilai kedua : "))
        div = i1 / i2
        print("Hasil pembagian adalah",div)
 
    elif choice == '5':
        print("Terimakasih, telah menggunakan kalkulator HELENA ROSALINA") 
        break
 
    else:
        print("Input anda salah, silahkan coba lagi ")
